models/transfer_check.go

```diff
@@ -145,16 +148,16 @@ type TransferCreateBody struct {
 	SkipScore    *bool
 }
 
-func (t TransferDataCreateBody) ToIngestionMap(id string) map[string]any {
+func (t TransferDataCreateBody) ToIngestionMap(mapping TransferMapping) map[string]any {
 	return map[string]any{
 		// there is a trap here: we map it to "object_id" to match what we do elsewhere on data model tables
-		"object_id": t.TransferId,
+		"object_id": ObjectIdWithPartnerIdPrefix(mapping.PartnerId, t.TransferId),
```

Comment:
because the object_id is the unique key for the purpose of identifying ingested objects in the org, and we want to guarantee partners cannot overwrite/read each other's ingested objects.

Reply:
so we must make object_id unique per partner in our code